### PR TITLE
Add help request creation page

### PR DESCRIPTION
**Pull request by Everkers**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -4,6 +4,6 @@ NEXT_PUBLIC_FIREBASE_PROJECT_ID=
 NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=
 NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=
 NEXT_PUBLIC_FIREBASE_APP_ID=
+NEXT_PUBLIC_MAP_BOX_ACESS_TOKEN=
```
Comment (soufyanelf) [inline .env.example:7]: why maxbox and not google map?

File: src/app/helps/create/components/RequestConfirmation/index.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                                alignItems: "center",
+                                marginRight: 7,
+                                fontWeight: "bold",
+                                color: "#0E6146",
+                                marginTop: 10,
+                                gap: 5,
+                              }}
+                            >
+                              <PlaceOutlined />
+                              <span>افتح خريطة الموقع</span>
```
Comment (soufyanelf) [inline src/app/helps/create/components/RequestConfirmation/index.js:94]: we will need this btn inside the help not here.

Comment (Everkers) [reply src/app/helps/create/components/RequestConfirmation/index.js:94]: I understand. It seems there was an unintended addition on the confirmation screen. Nonetheless, it's beneficial for UX, offering a full request preview and ensuring the location points correctly before the submission?

Comment (soufyanelf) [reply src/app/helps/create/components/RequestConfirmation/index.js:94]: make sense

**Review by soufyanelf — APPROVED** [review]

**Pull request merged**